recipes-soletta/soletta/soletta_git.bb

```diff
@@ -105,6 +105,9 @@ do_configure_append() {
        sed -i -e 's/^ *FLOW_NODE_TYPE_GTK=.*/# FLOW_NODE_TYPE_GTK is not set/g' ${S}/.config
    fi
 
+   sed -i "s/^\(CONFIG_LDFLAGS=\).*/\1\"${TARGET_LDFLAGS}\"/g" ${S}/.config
```

Comment:
what about you define TARGET_CFLAGS? or something like that and force those to go into  the build... I'm confused why the flags are unset if we should use them :-)